scheduler.py

```diff
@@ -196,18 +233,9 @@ def main():
             # someRefToSleepTimer.sleep(sleep_time)
             print(f"Temporary print replacing sleep: sleepTimer.sleep({str(sleep_time)})")
         else:
-            '''
-            TODO: Wesley
-
-            ensemble = active_ensembles["ensemble_list"][next_ensemble]
-
-            # perform ensemble functions
-            for (auto f : ensemble)
-            	std::invoke(f, inputs) # Wesley
-            	# TODO: allow for function arguments via e["inputs"]
-        		# should just be comma-separated list of inputs
+            print("performing ensemble functions: ")
+            perform_ensemble_functions(0) # TODO Iterate through the ensemble functions
```

Comment:
We should be performing the current ensemble specified in `active_ensembles.json`. Specifically, this should be in the `next_ensemble` variable.